lib/config.js

```diff
@@ -140,8 +141,31 @@ function getLocalConfig(thisConfig, directory) {
         config = ConfigOps.merge(localConfig, config);
     }
 
-    // Use the personal config file if there are no other local config files found.
-    return found || thisConfig.useSpecificConfig ? config : ConfigOps.merge(config, getPersonalConfig());
+    if (found || thisConfig.useSpecificConfig) {
+        return config;
+    } else {
+
+        /**
```

Comment:
Technically, I think `/**` is JSDoc notation, isn't it?  Should this perhaps just be `/*`?  Maybe it doesn't matter.